Makefile

```diff
@@ -129,6 +129,8 @@ test_serial: $(CONFIG) ## Runs RSpec and yarn tests serially
 
 fast_test: export RAILS_ENV := test
 fast_test: ## Abbreviated test run, runs RSpec tests without accessibility specs
+	ulimit -Sn 61440
```

Comment:
A question from my own ignorance: Will the `ulimit` configuration here persist in the developer's terminal after the test has concluded? (And if so, can we find a way to avoid that side effect?)

Reply:
Experimenting with it (using `ulimit -n` to view the configuration) the answer appears to be no; the ulimit applies only as the `make` command is running

Reply:
Any particular reason you chose the default hard limit of the mac os?  I guess the only issue i can think of is we might make the entire system unresponsive if the process does actually decide to use the limit or there is some type of errant process.  That's probably highly unlikely so I think this particular config is good to go...